maintenance/schemas/device-config.json

```
@@ -282,6 +282,9 @@
 				"treatBasicSetAsEvent": {
 					"const": true
 				},
+				"treatMultilevelSwitchAsEvent": {
```

```suggestion
				"treatMultilevelSwitchSetAsEvent": {
```
(rename the other occurences similarly)